documentation/docs/15-configuration.md

```diff
@@ -26,6 +26,7 @@ const config = {
 			}
 		},
 		env: {
+      devModes: ['development'],
```

Comment:
An 'useless' comment, but there seem to be a small indenting issue here (lacking 1 space)